edgelet/edgelet-utils/src/yaml_file_source.rs

```diff
@@ -116,8 +116,8 @@ fn from_yaml_value(uri: Option<&String>, value: Yaml) -> Result<Value, ConfigErr
 
             Ok(Value::new(uri, l))
         }
-
-        // 1. Yaml NULL
+        // 1. Yaml NULL - Option::None will transform into ValueKind::Nil
```

Comment:
Since `None` transforms into `Nil`, fix it on line 129 instead. And remove the paragraph on line 126.